domdiv/draw.py

```diff
@@ -62,167 +385,367 @@ def registerFonts(self):
-            #    +       +CC. . . . . . . . . . . . . . . . . .M+       +
-            #            |                                      |
-            #    +       +DD. . . . . . . . . . . . . . . . . .L+       +
-            #            |                                      |
-            #  FF+-------+EE                                   K+-------+J
-            #    |                                                      |
-            #    |                      Reverse Side                    |
-            #    |                       rotated 180                    |
-            #    |                                                      |
-            #   A+-------+B                                             +I
-            #            |                                              |
-            #    +      C+---------------------+D                 G+----+H
-            #                                  |                   |
-            #    +                            E+-------------------+F   +
-            #
-            #           delta x              delta y
-            delta = [(0, theTabHeight + notch_height),  # to A
-                     (notch_width1, 0),  # A  to B
-                     (0, -notch_height),  # B  to C
-                     (tab_2_notch, 0),  # C  to D
-                     (0, -theTabHeight),  # D  to E
-                     (theTabWidth, 0),  # E  to F
-                     (0, theTabHeight),  # F  to G
-                     (side_2_tab, 0),  # G  to H
-                     (0, notch_height),  # H  to I
-                     (0, body_minus_notches),  # I  to J
-                     (-notch_width2, 0),  # J  to K
-                     (0, notch_height),  # K  to L
-                     (0, stackHeight),  # L  to M
-                     (0, notch_height),  # M  to N
-                     (notch_width2, 0),  # N  to O
-                     (0, body_minus_notches),  # O  to P
-                     (0, notch_height),  # P  to Q
-                     (-side_2_tab, 0),  # Q  to R
-                     (0, stackHeight),  # R  to S
-                     (0, theTabHeight),  # S  to T
-                     (-theTabWidth, 0),  # T  to U
-                     (0, -theTabHeight),  # U  to V
-                     (0, -stackHeight),  # V  to W
-                     (-tab_2_notch, 0),  # W  to X
-                     (0, -notch_height),  # X  to Y
-                     (-notch_width1, 0),  # Y  to Z
-                     (0, -body_minus_notches),  # Z  to AA
-                     (notch_width3, 0),  # AA to BB
-                     (0, -notch_height),  # BB to CC
-                     (0, -stackHeight),  # CC to DD
-                     (0, -notch_height),  # DD to EE
-                     (-notch_width3, 0),  # EE to FF
-                     (0, -body_minus_notches)]  # FF to A
-
-            self.canvas.lines(DeltaXYtoLines(delta))
 
+            # Set up values used in the outline
```

Comment:
Also, maybe move the two collections of outline drawing calls to sub methods? This one has gotten pretty long...

Reply:
Not doing this for now.  In large part because there would be a lot of overlap in the setup of each.

But I have been thinking about ways to split between tab-only, regular divider, and wrapper/slip_case.  So basically, a unique part of drawDivider() for each of the three.